Lib/test/test_gzip.py

```diff
@@ -750,6 +763,35 @@ def test_compress_infile_outfile(self):
         self.assertEqual(out, b'')
         self.assertEqual(err, b'')
 
+    @add_compress_level_flag('--fast', '--best')
+    @create_and_remove_directory(TEMPDIR)
+    def test_compress_infile_outfile(self, compress_level):
+        local_testgzip = os.path.join(TEMPDIR, 'testgzip')
+        gzipname = local_testgzip + '.gz'
+        self.assertFalse(os.path.exists(gzipname))
+
+        with open(local_testgzip, 'wb') as fp:
+            fp.write(self.data)
+
+        rc, out, err = assert_python_ok('-m', 'gzip', compress_level, local_testgzip)
+
+        self.assertTrue(os.path.exists(gzipname))
+        self.assertEqual(rc, 0)
+        self.assertEqual(out, b'')
+        self.assertEqual(err, b'')
+
+    def test_compress_fast_best_are_exclusive(self):
+        rc, out, err = assert_python_failure('-m', 'gzip', '--fast', '--best')
+        self.assertIn(b"error: argument --best: not allowed with argument --fast", err)
+        self.assertGreater(rc, 0)
```

Comment:
Can't we expect a specific exit code? Same question for following test.

Reply:
+1